lightning/restore/tidb.go

```diff
@@ -115,15 +115,20 @@ func (timgr *TiDBManager) InitSchema(ctx context.Context, database string, table
 	return errors.Trace(err)
 }
 
-var createTableRegexp = regexp.MustCompile(`(?i)CREATE TABLE( IF NOT EXISTS)?`)
+var createTableRegexp = regexp.MustCompile(
```

Comment:
why not use restore feature of tidb parser？

Reply:
It's born after Lightning 😅